### ISSUE TITLE
 [a11y] series.stateMarkerGraphic missing after accessibility module import in v7.1.2

### ISSUE THREAD

**Issue by seintun** [highcharts/highcharts#15433]
#### Expected behaviour
Return `stateMarkerGraphic` when onMouse is interacted with the tracker.

#### Actual behaviour
`stateMarkerGraphic` is not returned when onMouse is interacted with the tracker.

#### Steps to reproduce

Have a few regression test suite that uses jQuery to simulate mouse interaction and validate the coordinations, but the following sample snippet failed to generate such `stateMarkerGraphic` after `v7.1.1`

```
const eventData = {
    pageX: coords.x,
    pageY: coords.y,
};

const series = hcChart.series[seriesIndex];
$(series.tracker.element).trigger($.Event('mousemove', eventData));
```

My config for series marker is marked as "enabled: false". But turning to true doesn't make a difference.
Is there a change on interacting with the `tracker.element` compared to `v7.1.1`?

Follow-up from https://www.highcharts.com/forum/viewtopic.php?f=9&t=45886

#### Product version

Following changes from Highcharts `v7.1.2`. Upgrading to latest version yields similar outcome. Works till `v7.1.1`

**Disabling/Commenting this code will return the expected `stateMarkerGraphic`**

https://github.com/highcharts/highcharts/blob/v7.1.2/js/modules/accessibility/components/SeriesComponent.js#L59

```
        if (options.marker && options.marker.enabled === false) {
            series.a11yMarkersForced = true;
            merge(true, series.options, {
                marker: {
                    enabled: true,
                    states: {
                        normal: {
                            opacity: 0
                        }
                    }
                }
            });
        }
```

Thanks!

I have attached an image of missing stateMarkerGraphic from codepen (left v7.1.2) VS visible stateMarkerGraphic from my working branch (right <= v7.1.1).

![Comparison_Img](https://user-images.githubusercontent.com/10414451/113190800-9c575300-9211-11eb-8e6b-a75c8fc046b9.jpg)

**Comment by KacperMadej** [comment]
Hi @seintun 

Thank you for reporting the problem.
Please add a live demo for the problem.

fyi: a11y @oysteinmoseng

**Comment by oysteinmoseng** [comment]
Hi @seintun,

`stateMarkerGraphic` is internal and not a public property, so this does not necessarily sound like a bug to me, but you might need to update your code to use a different property with the current Highcharts behavior.

If you can explain your use case, and like @KacperMadej mentions, provide a demo in CodePen/jsFiddle we might be able to help you out further.

**Comment by oysteinmoseng** [comment]
Closing this for now, feel free to comment for reopening.
